### ISSUE TITLE
Dependencies not found

### ISSUE THREAD

**Issue by esidebo** [ac-tuwien/pymhlib#16]
I was addressing this in #14 but did so poorly: I was being unclear and the issue was not a specific package. There seems to be some problem with the dependencies, they are not automatically installed when installing pymhlib with pip. Steps to reproduce:

```
* conda create --name test-pymhlib python=3.8 # Using a conda env
* conda activate test-pymhlib
* python -m pip install -U pymhlib # Does not install dependencies
```

**Comment by graidl** [comment]
I see now what you mean, thanks for pointing out this problem. Obviously, although requirements.txt exists, it is not used by setup. Will try to fix it asap!

**Comment by graidl** [comment]
Fixed in version 0.1.5, which has also been uploaded to PyPI.
